Adk-challenge-lab-guide.html

```
@@ -0,0 +1,1644 @@
+                            Searchを呼び出し、その結果を根拠（grounding）として回答を生成するようになります。
+                        </p>
+                    </div>
+
+                    <div class="diagram-wrap">
+                        <pre class="mermaid">
+sequenceDiagram
+    participant U as ユーザー
+    participant A as Travel Scout LlmAgent
+    participant G as google_search Tool
+
+    U->>A: "東京の2025年の主要イベントは"
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)